tests/test_gpu_stump.py

```diff
@@ -44,7 +44,7 @@ def test_gpu_stump_int_input():
 def test_gpu_stump_self_join(T_A, T_B):
     m = 3
     zone = int(np.ceil(m / 4))
-    ref_mp = naive.stamp(T_B, m, exclusion_zone=zone)
+    ref_mp = naive.stump(T_B, m, exclusion_zone=zone, row_wise=True)
```

Comment:
I think this looks great! Just by looking at the `naive.stump` input parameters it is obvious what is happening.

Reply:
Your are right! It is easy to understand what is going on.